testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/SettingsMenuActivity.kt

```diff
@@ -103,9 +107,12 @@ class SettingsMenuActivity : BaseActivity() {
         validateInputIDs()
     }
 
-    private fun createSettingsInfo(): String {
-        return "Build " + getString(R.string.miniapp_sdk_version) + " - " +
-                getString(R.string.build_version)
+    private fun createBuildInfo(): String {
+        return String.format(
```

Comment:
I guess it's personal preference, but I prefer this style of String templating because it's easier to understand what the final string will look like: 

```
val sdkVersion = getString(R.string.miniapp_sdk_version)
val buildVersion = getString(R.string.build_version)
return "Build $sdkVersion - $buildVersion"
```

But I don't have a strong opinion, so either way is okay with me.

Reply:
Yeah, this template is robust while reading. I have updated it. Thank you.